jormungandr/src/blockchain/multiverse.rs

```diff
@@ -43,12 +43,9 @@ impl<T: Clone> Multiverse<T> {
     }
 }
 
-impl Multiverse<Ledger> {
+impl<T> Multiverse<T> {
```

Comment:
Huh? I was pretty sure I already did that.

Reply:
Hmmm, probably you did it the same with the actual multiverse in chain-libs